stepconf/stepconf.go

```diff
@@ -160,6 +172,15 @@ func setField(field reflect.Value, value, constraint string) error {
 		field.SetFloat(f)
 	case reflect.Slice:
 		field.Set(reflect.ValueOf(strings.Split(value, "|")))
+	case reflect.Ptr:
+		// if the field is a ptr type then we need to initialize a pointer address first
+		// and then get the type the address points to and set a new non-nil zero type to it
+		var dePtrdType = field.Type().Elem()     // get ptr-less type of the field
+		var newPtrType = reflect.New(dePtrdType) // create new ptr address for type with non-nil zero value
```

Comment:
What if value is not set, should'nt then the value be nil?

Reply:
will be nil